src/status_im/ui/screens/browser/views.cljs

```diff
@@ -72,17 +73,21 @@
 
 (views/defview browser []
   (views/letsubs [webview (atom nil)
+                  open-new-url-error? (atom false)
                   {:keys [address]} [:get-current-account]
-                  {:keys [dapp? contact url] :as browser} [:get-current-browser]
-                  {:keys [can-go-back? can-go-forward?]} [:get :browser/options]
+                  {:keys [dapp? contact url browser-id] :as browser} [:get-current-browser]
+                  {:keys [can-go-back? can-go-forward? new?] :as opts} [:get :browser/options]
                   extra-js [:web-view-extra-js]
                   rpc-url [:get :rpc-url]
                   unread-messages-number [:get-chats-unread-messages-number]
                   network-id [:get-network-id]]
     [react/keyboard-avoiding-view styles/browser
      [status-bar/status-bar]
      [toolbar.view/toolbar {}
-      (toolbar.view/nav-back-count)
+      (if @open-new-url-error?
+        (toolbar.view/nav-button (toolbar.actions/back #(do (re-frame/dispatch [:navigate-back])
```

Comment:
it's also better to keep views 'dumb' without any logic, so better to move this logic to event `(toolbar.view/nav-button (toolbar.actions/back #(re-frame/dispatch [:browser-navigate-back])`